tools/preview/styles.css

```
@@ -0,0 +1,287 @@
+  padding: 6px 10px 6px 26px; position: relative; margin-bottom: 6px;
+}
+.warnings li::before { content: "!"; position: absolute; left: 11px; font-weight: 700; }
+.warnings li.ok { color: rgb(var(--text-tertiary)); background: transparent; border-color: transparent; }
+.warnings li.ok::before { content: "\2713"; color: #16a34a; }
+
+.prompt {
+  margin: 0; background: rgb(var(--bg-sunken));
+  border: 1px solid rgb(var(--border-subtle));
+  border-radius: var(--radius-md);
+  padding: 9px 11px; font-size: 11.5px; line-height: 1.5;
+  white-space: pre-wrap; word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)